about.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Code Coop</title>
+    <link rel="stylesheet" href="assets/css/about.css">
+  </head>
+  <body>
+    <header>
+        <h1>Code Coop</h1>
+    </header>
+      <nav>
+        <a href="index.html" title="Home page">About</a>
+        <a href="contact.html" title="Contact page with contact details">Contact Us</a>
+        <a href="#">Other</a>
```

Comment:
Do we need the "Other" link if we don't have a page yet?

Reply:
Latest commit has removed these links as they were just a placeholder.